src/web_ui/src/components/Pages/__tests__/Homepage.test.js

```diff
@@ -1,32 +1,23 @@
 import React from 'react';
-import { render, unmountComponentAtNode } from 'react-dom';
-import { act } from "react-dom/test-utils"
+import { configure, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import Album from "../Homepage";
```

Comment:
rename component to Homepage for consistency and to make it easier for other developers to understand